### ISSUE TITLE
Auto-complete on bride and groom status

### ISSUE THREAD

**Issue by dreams-togo** [FreeUKGen/ComETT#120]
As the options for entry are limited (mainly spinster, batchelor, widow, etc), could this field autocomplete once you start typing the start of the word. There may be other fields where this could apply too?

**Comment by BevStubbs** [comment]
I agree having an autocomplete would be advantageous as long as you can overtype something not on the autocomplete.
